grafana.sh

```diff
@@ -134,5 +134,48 @@ curl -X POST -H "Content-Type: application/json" \
   -d @/tmp/import_dashboard.json \
   http://localhost:3000/api/dashboards/db
 
+# Install Infinity Datasource Plugin
+INFINITY_PLUGIN_VERSION="2.9.5"
+echo "Installing Infinity Datasource plugin..."
+grafana-cli plugins install yesoreyeram-infinity-datasource $INFINITY_PLUGIN_VERSION
+sudo systemctl restart grafana-server
+
+# Configure the Infinity datasource
+echo "Configuring Infinity data source..."
+curl -X POST -H "Content-Type: application/json" \
+  -u $GRAFANA_ADMIN_USER:$GRAFANA_ADMIN_PASSWORD \
+  -d '{
+    "name": "yesoreyeram-infinity-datasource",
+    "type": "yesoreyeram-infinity-datasource",
+    "access": "proxy"
+  }' \
+  http://localhost:3000/api/datasources
+
+# Fetch and prepare the znnd dashboard
+ZNND_DASHBOARD_JSON_URL="https://raw.githubusercontent.com/go-zenon/go/main/dashboards/znnd.json"
```

Comment:
if we host these scripts somewhere, then we should host the dashboard alongside it as well